Python/Environment_Variables/env.py

```diff
@@ -0,0 +1,42 @@
+'''
```

Comment:
We don't need a script docstring (I cant recall the exact name), either put this in README. or get rid of it all together

Reply:
Done @vybhav72954